src/test/resources/api/__files/body-search-user-repos-O8W78.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
is there a corresponding test for these json files?

Reply:
I was too fast to resolve the last comment regarding test data and I cannot find the comment anymore. Anyhow, to answer your question, yes, the file(s) is used in these tests:
`org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigatorTest#fetchRepos_BelongingToAuthenticatedUser_FilteredByTopic`
`org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigatorTest#fetchRepos_BelongingToAuthenticatedUser_FilteredByTopic_RemovesAll` and `org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigatorTest#fetchRepos_BelongingToAuthenticatedUser_FilteredByMultipleTopics`. 

Here is the new expected url with the query parameters:
```
{
  "request" : {
    "urlPath" : "/search/repositories",
    "method" : "GET",
    "basicAuth": {
      "username": "git-user",
      "password": "git-secret"
    },
    "queryParameters" : {
      "q" : {
        "equalTo" : "topic:awesome org:stephenc"
      }
    }
  },
```